lib/rules/space-in-parens.js

```diff
@@ -181,23 +175,35 @@ module.exports = {
             }
 
             if (ALWAYS) {
-                return isOpenerException(right);
+                return isOpenerException(checkPunctuator(right, left));
             }
-            return !isOpenerException(right);
-
+            return !isOpenerException(checkPunctuator(right, left));
         }
 
         /**
-         * Determines if an closer paren should not have an existing space after it
+         * Determines if a closing paren has a space immediately before it
          * @param {Object} left The token before the paren
          * @param {Object} right The paren token
-         * @returns {boolean} True if the paren should reject the space
+         * @returns {boolean} True if the paren is missing the required space
          */
-        function shouldCloserRejectSpace(left, right) {
-            if (astUtils.isOpeningParenToken(left)) {
+        function closerMissingSpace(left, right) {
```

Comment:
While we're in this space, could we improve the parameter names? Names like `tokenBeforeClosingParen` and `closingParenToken` would be much easier to read than `left` and `right`.